firstvoices/backend/tests/test_apis/test_sites_api.py

```diff
@@ -186,3 +186,142 @@ def test_detail_404(self):
         response = self.client.get(f"{self.get_detail_endpoint('fake-site')}")
 
         assert response.status_code == 404
+
+
+class TestMySites(BaseApiTest):
+    """
+    End to end tests that check the my-sites endpoint for expected behavior.
+    """
+
+    API_LIST_VIEW = "api:my-sites-list"
+    APP_NAME = "backend"
+
+    @pytest.mark.django_db
+    def test_list_empty(self):
```

Comment:
Based on the changes to the base API test class I think that this test can be removed

Reply:
Nevermind, I see this is a different case